commands/init.py

```diff
@@ -0,0 +1,52 @@
+import logging
+import datetime
+
+from config import config_context
+
+help="Interactively initialize a new configuration"
```

Comment:
It follows the style used in other modules, by which I mean I copied it from another file